environment.yaml

```
@@ -0,0 +1,15 @@
+channels:
+  - conda-forge
```

Maybe include `anaconda` channel since it is used in several packages.
```suggestion
  - conda-forge
  - anaconda
```